acos_client/v21/slb/server.py

```diff
@@ -22,11 +22,12 @@ class Server(base.BaseV21):
     def get(self, name, **kwargs):
         return self._post("slb.server.search", {'name': name}, **kwargs)
 
-    def create(self, name, ip_address, **kwargs):
+    def create(self, name, ip_address, conn_limit=8000000, **kwargs):
         params = {
             "server": {
                 "name": name,
                 "host": ip_address,
+                "conn-limit": conn_limit,
```

Comment:
Since the conn-limit isn't required by acos we shouldn't be sending it when it's not specified.